extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -345,18 +377,27 @@ export class Bind {
   apply_(results) {
     this.boundElements_.forEach(boundElement => {
       const {element, boundProperties} = boundElement;
+      const tagName = element.tagName;
 
       this.resources_.mutateElement(element, () => {
         const mutations = {};
         let width, height;
 
         boundProperties.forEach(boundProperty => {
-          const expressionString = boundProperty.expressionString;
-          const newValue = results[expressionString];
+          const {property, expressionString, previousResult} =
+              boundProperty;
+
+          // Rewrite attribute value if necessary.
+          // This is not done in the worker since it relies on `url#parseUrl`,
```

Comment:
The problem is that [`parseUrl` uses `document.createElement`](https://github.com/ampproject/amphtml/blob/master/src/url.js#L54), which IIUC is unavailable to workers.

To workaround, we could either change this implementation to not use DOM or even use `Worker#postMessage` to delegate this to main page. Thoughts?